scripts/ecs-anywhere-install.sh

```diff
@@ -592,6 +600,75 @@ wait-agent-start() {
     fail
 }
 
+exec-setup() {
+    find-and-move-certs
+    install-ssm-binaries-for-exec
+}
+
+find-and-move-certs() {
+    CERTS_PATH="/var/lib/ecs/deps/execute-command/certs"
+    CERTS_FILES_LOCATIONS=("/etc/ssl/certs/ca-certificates.crt"
+      "/etc/pki/tls/certs/ca-bundle.crt"
+      "/etc/ssl/ca-bundle.pem"
+      "/etc/pki/tls/cacert.pem"
+      "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
+      "/etc/ssl/cert.pem"
+    )
+
+    echo "Copying certs for exec"
+    certs=""
+    if [ -n "$CERTS_FILE" ] && [ -f "$CERTS_FILE" ]; then
+        certs=$CERTS_FILE
+    else
+        if [ -n "$CERTS_FILE" ]; then
+            echo "Provided certs file does not exist, looking for certs in known possible locations"
+        fi
+        for f in "${CERTS_FILES_LOCATIONS[@]}"
+        do
+            if  [ -f "$f" ]; then
+                certs=$f
+                break
+            fi
+        done
+    fi
+
+    if [ -z "$certs" ]; then
+        echo "Could not find certificates. Please generate TLS certificates, or provide a valid file path to them"
+        fail
+    else
+        echo "Using $certs"
+        mkdir -p $CERTS_PATH && cd $CERTS_PATH
+        cp "$certs" tls-ca-bundle.pem
+        chmod 400 tls-ca-bundle.pem
+    fi
+    ok
+}
+
+install-ssm-binaries-for-exec() {
+    BINARY_VERSION="3.1.338.0"
+    BINARY_PATH="/var/lib/ecs/deps/execute-command/bin/${BINARY_VERSION}"
+    ARCHITECTURE="$(uname -m)"
+
+    echo "Downloading SSM binaries for exec feature"
+    cd && mkdir -p ssm-binaries && cd ssm-binaries
+    case $ARCHITECTURE in
+        'x86_64')
+            curl https://s3.${REGION}.amazonaws.com/amazon-ssm-${REGION}/${BINARY_VERSION}/linux_amd64/amazon-ssm-agent-binaries.tar.gz -o amazon-ssm-agent.tar.gz
```

Comment:
For CN partition, I think, this S3 url will be a bit different -> s3.${REGION}.amazonaws.com.cn